### DISCUSSION TITLE
kenapa ga bisa seed ni

### DISCUSSION THREAD

**Discussion by falfikri** [imamsudarajat04/nongkrongkoding#37] [Q&A] [answered]
kenapa ga mau php artisan db:seed ni, atau harus pake class ?

**Comment by imamsudarajat04** [comment, marked as answer]
itu udah masuk datanya, cuman karna coloumnnya foreignkey jadi duplicate, abaikan aja data udah masuk

**Reply by falfikri** [reply]
ooh, ntar aku cek
